src/test/java/uk/gov/companieshouse/mapper/OtherReasonMapperTest.java

```diff
@@ -35,10 +44,16 @@ void shouldReturnNullWhenValueIsNull() {
 
         @Test
         void shouldMapValueWhenValueIsNotNull() {
-            OtherReasonEntity mapped = mapper.map(new OtherReason(title, description, Collections.emptyList()));
+            List<Attachment> attachments = new ArrayList<Attachment>();
+            attachments.add(mockAttachment);
+
+            OtherReasonEntity mapped = mapper.map(new OtherReason(title, description, attachments));
+
             assertEquals(title, mapped.getTitle());
             assertEquals(description, mapped.getDescription());
-            assertTrue(mapped.getAttachments().isEmpty());
+            assertEquals(mockAttachment, attachments.get(0));
```

Comment:
This is incorrect as you're testing the list you added the mock attachment to, not the attachment list from made by the mapper.
Remember to setup the when for the AttachmentMapper

Reply:
Done!